eth/scripts/1_setup_bridge_contracts.js

```diff
@@ -92,17 +85,6 @@ async function main() {
     );
   }
 
-  // --- add Aleph -> Ethereum token pairs
```

Comment:
why this change?

Reply:
We removed this pair when we merged change https://github.com/Cardinal-Cryptography/most/commit/f2bedb59fd98f884ddd4909c29484122508ba81e, so that one must brigde native AZERO with `sendRequestAzeroToNative`